style.css

```diff
@@ -235,7 +251,7 @@ body {
 
 .project {
   width: fit-content;
-  margin-top: 20px;
+  margin-top: 60px;
```

Comment:
- [ ] Kindly remove or reduce the `margin-top` in this class to match the Figma design. 
![image](https://user-images.githubusercontent.com/62290545/161156399-57baeacf-8e1c-41a7-a585-457bbf395e98.png)